draft/index.html

```diff
@@ -633,6 +633,7 @@ <h3>A Note on Candidate Retrieval and Scoring</h3>
 	</p>
 	<p>
 	  Deciding on a scoring method is one of the main difficulties in developing a reconciliation service.
+          Selecting candidates does not always rely on fuzzy matching. In those cases of exact match the score is an irrelevant information as candidates are either in or out. Thus the score attribute is optional.
```

Comment:
I feel like 'fuzzy matching' is confusing terminology here. E.g. a traditional search engine with basic TF-IDF scoring would not be considered 'fuzzy matching' in my opinion. I think it's a different level, e.g. when computing a score for a candidate, we can match individual strings with exact or fuzzy matching. To avoid that, maybe we could say something like this:

```suggestion
          Depending on the use case, we might not want a score at all: in cases like geo containment (a place should be inside a specific area) or date ranges (a point in time should be inside a specific time range) the score is irrelevant, as candidates are either in or out. Thus the score attribute is optional.
```

Reply:
I like your formulation, let's use it.